README.md

```diff
@@ -1,2 +1,83 @@
-# lockstep-sdk-python
-Lockstep Platform API software development kit for Python
+# Fetch Invoice Sample Python Project
+
+Many types of products examine invoices for a customer and provide feedback on them. A typical product might analyze incoming invoices and add metadata like a **credit score** for each invoice. This tutorial explains how to iterate through invoices and examine them. 
```

Comment:
Can we please add in the shields.io status for this SDK?

Here's the markdown:

![PyPI](https://img.shields.io/pypi/v/lockstep-sdk)

```
![PyPI](https://img.shields.io/pypi/v/lockstep-sdk)
```

Reply:
We should probably have the top of the project be the "What is this repository?" and the second part is "Here's how to write a program using this SDK"